corehq/apps/domain/views/settings.py

```diff
@@ -391,11 +391,11 @@ def post(self, request, *args, **kwargs):
         else:
             disable_case_search(self.domain)
 
-        CaseSearchConfig.objects.update_or_create(domain=self.domain, defaults={
+        config, _ = CaseSearchConfig.objects.update_or_create(domain=self.domain, defaults={
             'enabled': request_json.get('enable'),
```

Comment:
`enabled` doesn't need to be taken out?

Reply:
no, `enabled` is just a field on the `CaseSearchConfig` model. The others were referencing related models through a many-to-many relationship.